pyaedt/application/AnalysisMaxwellCircuit.py

```diff
@@ -0,0 +1,51 @@
+from pyaedt.modeler.Circuit import ModelerMaxwellCircuit
+from pyaedt.application.Analysis import Analysis
+
+
+class AnalysisMaxwellCircuit(Analysis):
+    """Class for Maxwell Circuit (MaxwellCircuit)
+
+    Maxwell Circuit Editor has no setup, solution, analysis or postprocessor
+    It is automatically initialized by Application call (Maxwell Circuit).
+    Refer to Application function for inputs definition
+
+    Parameters
+    ----------
```

Comment:
@sushobhit-gupt Please add the parameters from the `__init__` here.

Reply:
added the list of parameters